examples_test.go

```diff
@@ -9,13 +10,13 @@ func Example() {
 	// flags.csv contains comma-separated flag names and sample rates.
 	// See: fixtures/flags_example.csv
 	backend := BackendFromFile("flags.csv")
-	Init(30*time.Second, backend)
+	_, ctx := Init(context.Background(), 30*time.Second, backend)
```

Comment:
Context is generally intended to be request-level, which means that each request would require initializing separately (and therefore a separate backend and ticker).

I don't think we actually need to pass a fresh context into the initialization, though - we could use the context to override and/or cache the values, but still otherwise creating the backend outside of the request lifecycle.